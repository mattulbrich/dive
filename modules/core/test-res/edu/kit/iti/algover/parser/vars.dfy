method manyVars() {

  var i: int;

  var x1, x2, x3: int;

  var y : int := 42;

  var v := 44;

}
