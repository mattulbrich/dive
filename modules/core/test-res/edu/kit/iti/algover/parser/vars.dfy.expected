(COMPILATION_UNIT
  (method manyVars ARGS
  (BLOCK
     (var i (TYPE int))
     (var x1 (TYPE int))
     (var x2 (TYPE int))
     (var x3 (TYPE int))
     (var y (TYPE int) 42)
     (var v 44))))
